src/db.js

```diff
@@ -1,2 +1,31 @@
 // database logic goes here
+const Product = require('../models/movie');
+const mongoose = require('mongoose');
+var app = require('../index')
```

Comment:
don't need this (you get `app` through your `setup` function). Might cause issues, please delete.